### PR TITLE
add view files

### PR DESCRIPTION
**Pull request by mosidrum**
In this PR: 

- I Implemented the design from the sneak peek wireframes

### CODE REVIEW

File: app/views/shared/_user.html.erb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<%= stylesheet_link_tag('index') %>
+<div class="container">
+  <div class="user">
+    <img src= <% user.photo %> alt="avatar.jpg"/>
```
Comment (KanzaTahreem) [inline app/views/shared/_user.html.erb]: - [ ] _Well done for using partials 👏👏_ However, despite of using the correct URL for photo, the image is not being displayed. Kindly make sure to use `<%= %>` tags to embed. Kindly fix this. Also please make sure to follow the [given wireframe](https://raw.githubusercontent.com/microverseinc/curriculum-rails/main/blog-app/images/blog_root_users.png?token=GHSAT0AAAAAAB5V373VYYE6N7XFP65YVMMUZIWC27Q) for styling

This is what I have in the terminal 👇
image

That's how it's currently displayed 👇
image

File: app/views/users/show.html.erb
Comment (KanzaTahreem) [inline app/views/users/show.html.erb:1]: - [ ] Great effort so far ✨ However, I have noticed that the current design is not according to the wireframe. Kindly make sure that the user show page is styled according to the [given wireframe](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_user_page.png?raw=true). You can add your styles but the general layout should be according to the given wireframe as per [this requirement](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Views_project.md#:~:text=Implement%20the%20design%20from%20the%20sneak%20peek%20wireframes.) ⛏🤓 This is what we have right now 👇

image

File: app/views/posts/index.html.erb
Comment (KanzaTahreem) [inline app/views/posts/index.html.erb:1]: - [ ] As highlighted in the above comment 👆 , kindly make sure to follow the [given wireframe](https://raw.githubusercontent.com/microverseinc/curriculum-rails/main/blog-app/images/blog_user_all_posts.png?token=GHSAT0AAAAAAB5V373VHWTNIXQCD3TKI3QUZIWDGDQ) for styling as per [this project requirement](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Views_project.md#:~:text=Implement%20the%20design%20from%20the%20sneak%20peek%20wireframes.). This s what we have right now 👇✨

image

File: app/views/posts/show.html.erb
Comment (KanzaTahreem) [inline app/views/posts/show.html.erb:1]: - [ ] As mentioned above, kindly make sure to follow the wireframe for styling. Please note that you can add your styles to it keeping the general layout of the [given wireframe](https://raw.githubusercontent.com/microverseinc/curriculum-rails/main/blog-app/images/blog_single_post.png?token=GHSAT0AAAAAAB5V373U7DF4L5GEOFL5WMIWZIWDJMA) as per [this requirement](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Views_project.md#:~:text=Implement%20the%20design%20from%20the%20sneak%20peek%20wireframes.) ⛏🤓

image

**Review by KanzaTahreem — CHANGES_REQUESTED** [review]
# Changes Required 🔁

Hi @mosidrum 

Good job so far 👏! I really appreciate the effort you put in!  **_However, there are few this that still need to be fixed. But you are almost there!_** 💪 🏁

so close


## Highlights 🎯

There are some excellent points that you have made perfectly. Keep up the great work! 👏
- ✅ Excellent use of partials
- ✅ Descriptive PR and README file
- ✅ Neat and clean code

## Required Changes 🛑

_Please check the comments under the review._

## Optional suggestions ❕

- [OPTIONAL} Please make sure to use descriptive commit messages - each commit is small enough to describe it in one sentence.💪 Also, kindly ensure that the commit messages are written in the imperative tense and capitalized. [Read more about how to write descriptive commit messages](https://www.freecodecamp.org/news/level-you-up-to-awesome-commit-messages-a85558cb90e8).

Here are some examples of descriptive commits:

image


_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👨‍💻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by mosidrum** [comment]
I have been waiting for 3hrs now

**Review by BregornOriginal — APPROVED** [review]
Hi @mosidrum 

Your project is complete!  There is nothing else to say other than... it's time to merge it 

Congratulations! 

giphy

## Highlights

✅ Linters are passing
✅ Excellent commit history
✅ Good work implementing changes requested by the previous reviewer 💯 

## Optional suggestions
Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.
Cheers and Happy coding!:aplauso::aplauso::aplauso:
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

Please, remember to tag me in your question so I can receive the notification.
------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)[.](https://airtable.com/shrQAqnBwek5a0O0s)._)

**Pull request merged**